blocks/main.go

```diff
@@ -221,7 +220,7 @@ func setupMongo(parsedURL *url.URL) (database, error) {
 	}
 
 	session.SetMode(mgo.Monotonic, true)
-	// session.SetSafe(&mgo.Safe{FSync: true})
+	session.SetSafe(&mgo.Safe{FSync: true})
```

Comment:
Nope. Just experimentally. It destroy MongoDB performance. I'll comment it out again.